samples/net/sockets/http_get/Makefile.posix

```diff
@@ -1,4 +1,10 @@
 # This makefile builds the sample for a POSIX system, like Linux
 
+INC = -I$(ZEPHYR_BASE)/include
```

Comment:
Are we still maintaining Makefiles?

Reply:
Maintaining? No. They just work ;-).

P.S. This Makefile is for POSIX, not for Zephyr. Obviously, there're no cmakes and ninjas in POSIX.